azure-functions/acbs-function/.eslintrc.js

```diff
@@ -18,7 +18,15 @@ const baseRules = {
   'import/no-extraneous-dependencies': [
     'error',
     {
-      devDependencies: ['**/*.test.{js,ts}', '**/*.api-test.{js,ts}', '**/*.spec.{js,ts}', '**/webpack.*.{js,ts}', '**/api-test*/**', '**/__mocks__/**'],
+      devDependencies: [
+        '**/*.test.{js,ts}',
+        '**/*.ff-test.{js,ts}',
```

Comment:
Have added `.ff-test.` to this array - noticed this was missing when importing `import { createMocks } from 'node-mocks-http';`